src/main/java/org/sonarlint/intellij/promotion/PromotionProvider.kt

```diff
@@ -144,6 +148,10 @@ class PromotionProvider(private val project: Project) {
         }
     }
 
+    private fun processCICIDProjectAnalysisPromotion(notifications: SonarLintProjectNotifications) {
+        showPromotion(notifications, "Speed up the CI/CD analysis")
```

Comment:
The text is not right here, SC/SQ won't speed up the CI/CD, the contrary in fact :)
I think the argument should be "Analyze your project in your CI/CD pipeline"